lib/improver/nbhood.py

```diff
@@ -135,31 +104,33 @@ def find_required_lead_times(cube):
         return required_lead_times
 
     @staticmethod
-    def get_grid_x_y_kernel_ranges(cube, radius_in_km, max_kernel_cell_radius):
+    def get_neighbourhood_width_in_grid_cells(
+            cube, radius_in_km, max_radius_in_grid_cells):
         """
         Return the number of grid cells in the x and y direction
-        to be used to create the kernel.
+        used to define the neighbourhood width in the x and y direction
+        based on the input radius in km.
 
         Parameters
         ----------
         cube : Iris.cube.Cube
             Cube containing the x and y coordinates, which will be used for
             calculating the number of grid cells in the x and y direction,
-            which equates to the size of the desired radius.
+            which equates to the size of the desired radii.
```

Comment:
In this case, radius_in_km is singluar, so the comment was correct with "radius", not "radii".